onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -902,7 +894,10 @@ def main():
                 decoder_merged_model_fp32_opt_path,
             ]
 
-            # Run the optimizer script
+            # Run the optimizer script, runs the torch as well.
+            if not args.use_dynamo_export:
```

Comment:
This `if` case's logic does not match the suggestion [here](https://github.com/microsoft/onnxruntime/pull/21713/files#r1717660144).